### PR TITLE
update event when drag and resize

### PR DESCRIPTION
**Pull request by vietbq**

### CODE REVIEW

**Comment by daodat18plus** [comment]
OK

**Comment by limkimhuor** [comment]
ok

File: app/assets/javascripts/calendar.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
-  var start_time, end_time, event_title;
-
+  var start_date, finish_date, event_title;
+
```
Comment (dieunb) [inline app/assets/javascripts/calendar.js]: remove trailing space

File: app/assets/javascripts/calendar.js [hunk 2]
```diff
@@ -70,9 +70,15 @@ $(document).ready(function() {
...
+    },
+    eventResize: function(event, delta, revertFunc) {
+      updateEvent(event);
+    },
+    eventDrop: function(event, delta, revertFunc) {
+      updateEvent(event);
     }
   });
-
+
```
Comment (dieunb) [inline app/assets/javascripts/calendar.js]: remove trailing space

**Pull request merged**